lnbits/wallets/eclair.py

```diff
@@ -70,10 +70,8 @@ async def create_invoice(
         **kwargs,
     ) -> InvoiceResponse:
 
-        data: Dict = {
+        data: Dict[str, Any] = {
             "amountMsat": amount * 1000,
-            "description_hash": b"",
-            "description": memo,
```

Comment:
- previously the `description` was always sent
- now `description` is sent only if both `description_hash` and `unhashed_description` are missing